woodwork/tests/schema/test_table_schema_init.py

```diff
@@ -239,16 +240,16 @@ def test_schema_with_numeric_time_index(sample_column_names, sample_inferred_log
 
 def test_schema_init_with_logical_type_classes(sample_column_names, sample_inferred_logical_types):
     logical_types = {
-        'full_name': NaturalLanguage,
+        'full_name': Unknown,
```

Comment:
I think we can leave this as `NaturalLanguage` so we know it was initialized properly using the specified type instead of defaulting to `Unknown`. Not sure why we used `NaturalLanguage` here before...

Reply:
done!